lnbits/settings.py

```diff
@@ -1,92 +1,298 @@
+    lnbits_reserve_fee_percent: float = Field(default=1.0)
+    lnbits_service_fee: float = Field(default=0)
+    lnbits_hide_api: bool = Field(default=False)
+    lnbits_denomination: str = Field(default="sats")
 
-RESERVE_FEE_MIN = env.int("LNBITS_RESERVE_FEE_MIN", default=2000)
-RESERVE_FEE_PERCENT = env.float("LNBITS_RESERVE_FEE_PERCENT", default=1.0)
-SERVICE_FEE = env.float("LNBITS_SERVICE_FEE", default=0.0)
+    # funding sources
+    lnbits_backend_wallet_class: str = Field(default="VoidWallet")
+    lnbits_allowed_funding_sources: List[str] = Field(
+        default=[
+            "VoidWallet",
+            "FakeWallet",
+            "CLightningWallet",
+            "LndRestWallet",
+            "LndWallet",
+            "LntxbotWallet",
+            "LNPayWallet",
+            "LNbitsWallet",
+            "OpenNodeWallet",
+            "LnTipsWallet",
+        ]
+    )
+    fake_wallet_secret: str = Field(default="ToTheMoon1")
+    lnbits_endpoint: str = Field(default="https://legend.lnbits.com")
+    lnbits_key: Optional[str] = Field(default=None)
+    cliche_endpoint: Optional[str] = Field(default=None)
+    corelightning_rpc: Optional[str] = Field(default=None)
+    eclair_url: Optional[str] = Field(default=None)
+    eclair_pass: Optional[str] = Field(default=None)
+    lnd_rest_endpoint: Optional[str] = Field(default=None)
+    lnd_rest_cert: Optional[str] = Field(default=None)
+    lnd_rest_macaroon: Optional[str] = Field(default=None)
+    lnd_rest_macaroon_encrypted: Optional[str] = Field(default=None)
+    lnd_cert: Optional[str] = Field(default=None)
+    lnd_admin_macaroon: Optional[str] = Field(default=None)
+    lnd_invoice_macaroon: Optional[str] = Field(default=None)
+    lnd_grpc_endpoint: Optional[str] = Field(default=None)
+    lnd_grpc_cert: Optional[str] = Field(default=None)
+    lnd_grpc_port: Optional[int] = Field(default=None)
+    lnd_grpc_admin_macaroon: Optional[str] = Field(default=None)
+    lnd_grpc_invoice_macaroon: Optional[str] = Field(default=None)
+    lnd_grpc_macaroon: Optional[str] = Field(default=None)
+    lnd_grpc_macaroon_encrypted: Optional[str] = Field(default=None)
+    lnpay_api_endpoint: Optional[str] = Field(default=None)
+    lnpay_api_key: Optional[str] = Field(default=None)
+    lnpay_wallet_key: Optional[str] = Field(default=None)
+    lntxbot_api_endpoint: Optional[str] = Field(default=None)
+    lntxbot_key: Optional[str] = Field(default=None)
+    opennode_api_endpoint: Optional[str] = Field(default=None)
+    opennode_key: Optional[str] = Field(default=None)
+    spark_url: Optional[str] = Field(default=None)
+    spark_token: Optional[str] = Field(default=None)
+    lntips_api_endpoint: Optional[str] = Field(default=None)
+    lntips_api_key: Optional[str] = Field(default=None)
+    lntips_admin_key: Optional[str] = Field(default=None)
+    lntips_invoice_key: Optional[str] = Field(default=None)
+
+    # boltz
+    boltz_network: str = Field(default="main")
```

Comment:
Configs for extensions must not be part of `core`.
Changes to an extension (like adding a new config param) should not require changes in `core`.
This sets a bad precedent.
Can the extension have its own `Admin UI` (a simple dialog where values are manipulated)?

Reply:
> Can the extension have its own Admin UI (a simple dialog where values are manipulated)?
very good idea, maybe this one we can add a little bit later, to get this PR done faster?
extension settings are still in main right now.